tests/unit/backend/models/ExportModelTest.php

```diff
@@ -48,4 +57,24 @@ public function testEncodeArrayValue()
         $this->assertEquals('art direction-roman empire-sci\-fi', $result);
     }
 
+    public function testExportContentType()
+    {
+        $model = new ExampleExportModel;
+
+        $csvName = $model->export(['foo' => 'title', 'bar' => 'title2'], []);
+
+        $response = $model->download($csvName);
+
+        $requestMock = $this
+            ->getMockBuilder('Illuminate\Http\Request')
+            ->setMethods()
+            ->getMock();
+        $response->prepare($requestMock);
+
+        $this->assertTrue($response->headers->has('Content-Type'), "Response is missing the Content-Type header!");
+        $this->assertTrue($response->headers->contains('Content-Type', 'text/plain'), "Content-Type is not \"text/plain\"!");
+
+        @unlink(temp_path() . '/' . $csvName);
```

Comment:
Is this required since download() should be deleting the file?

Reply:
Exactly.

I wanted to avoid unnecessary side-effects, so I tried to use the minimal amount of function calls. Since I know exactly what unlink does, I figure it would be safer.

Reply:
Isn't there a way to mock the send? I'd like to be able to verify that the file was automatically deleted after making the request

Reply:
I think we could just call `->send()` on the response. But I'm not that familiar with tests in October/Laravel, maybe there is a more elegant solution.

However, I'd suggest to write a separate test for this…

Reply:
@cleverer what's the rationale for the second test instead of just calling send()?

Reply:
Well it is a unit test, so it should test each piece of functionality separately, right? One test checking the headers being set right and one checking the file getting deleted afterwards. I think it is easyer to work with tests that just test a small and well defined piece of code.

But if the tests for October usually are more generalized, I'm happy to follow the lead and make them consistent with the others.

Reply:
In my mind we're testing the proper functionality of the export, which includes correct headers and cleaning up after itself, so you might need to change the name of the test method to reflect that but I think it should be one piece.